gradle/gson-deserializers-okhttp-asyncTask/app/src/main/java/com/irontec/kotlintest/MainActivity.kt

```diff
@@ -0,0 +1,83 @@
+package com.irontec.kotlintest
+
+import android.os.AsyncTask
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.util.Log
+import android.view.Menu
+import android.view.MenuItem
+import android.widget.TextView
+import com.google.gson.Gson
+import com.google.gson.GsonBuilder
+import com.irontec.kotlintest.deserializers.WeatherDeserializer
+import com.irontec.kotlintest.models.WeatherObject
+import com.irontec.kotlintest.networking.NetworkClient
+import org.json.JSONObject
+import java.io.BufferedInputStream
+import java.io.BufferedReader
+import java.io.InputStreamReader
+import java.net.HttpURLConnection
+import java.net.URL
+
+
+public class MainActivity : AppCompatActivity() {
+
+    open val mTextView: TextView
+        get() {
+            return findViewById(R.id.text) as TextView
```

Comment:
This is ok though quite strange, as `findViewById()` will be invoked on every property value get.
You can use [`Delegates.notNull`](http://kotlinlang.org/docs/reference/delegated-properties.html#lazy) here.